core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/withEditor.ts

```diff
@@ -207,7 +207,15 @@ export function withEditor() {
                     };
                 }),
                 $iframeURL: computed<string | InstanceType<typeof String>>(() => {
+                    /*
+                        Here we need to import pageAPIResponse() to create the computed dependency and have it updated every time a response is received from the PageAPI.
+                        This should change in future UVE improvements.
+                        More info: https://github.com/dotCMS/core/issues/31475 and https://github.com/dotCMS/core/issues/32139
+                     */
+                    const pageAPIResponse = store.pageAPIResponse();
+                    const vanityURL = pageAPIResponse?.vanityUrl?.url;
                     const isTraditionalPage = untracked(() => store.isTraditionalPage());
+                    const params = untracked(() => store.pageParams());
```

Comment:
Is just me or we been having this issue a couple of time already? or at least something around the iframeURL state?